src/main/groovy/com/marklogic/gradle/task/roxy/scaffoldTemplates/SampleTestFiles.groovy

```diff
@@ -0,0 +1,158 @@
+package com.marklogic.gradle.task.roxy
+
+class SampleTestFiles {
+
+    static String getSampleTestsXqy() {
+        return """
+xquery version '1.0-ml';
+
+import module namespace c = 'http://marklogic.com/roxy/test-config' at '/test/test-config.xqy';
+import module namespace test = 'http://marklogic.com/roxy/test-helper' at '/test/test-helper.xqy';
+import module namespace sec= 'http://marklogic.com/xdmp/security' at  '/MarkLogic/security.xqy';
+import module namespace cfg = 'http://thieme.com/synone/config' at '/app/config/config.xqy';
+
+
+declare option xdmp:mapping 'false';
+
+(: using global variable from app/config/config.xqy :)
+
+declare variable \\\$admin-user as xs:string := \\\$cfg:SEARCH-USER;
+declare variable \\\$admin-password as xs:string := \\\$cfg:SEARCH-PWD;
+
+
+xdmp:log('\$testName Tests BEGINNING.....')
+,
+(: using trace API - you can turn activate and deactivate this in the admin console :)
+xdmp:trace('ps-\$testName','Running in database ' || xdmp:database-name(xdmp:database()))
+,
+xdmp:trace('ps-\$testName','admin user =' || \\\$admin-user )
+
+,
+
+(: ***************************************************************
+
+Working with Roxy Unit Testing 
+
+Debugging: The first case to debug is where you have coded invalid or bad
+xquery. You may or may not be notified of this in the web interface.
+
+1. it is always good to pack log/trace statements into your tests.
+2. Keep a tail on the ML server log at all times - sometimes setup/teardown
+   will just fail without saying why (invalid xqy syntax). You will see this in the log.
+3. You may want to look at the state of the test db after the test has failed 
+   but by default it cleans up everything. So you cannot. On the test
+   UI disable the teardown modules by clicking away the appropriate tich
+   boxes. Rerun your tests and then you can examine left over docs in 
+   Query Console
+4. Keep it in small chunks : Code a little, test a little.
+5. There are a few different kind of assertions you can code. THey 
+   are defined in /test/test-helper.xqy
+6. I like the assertion function below becase as well as the condition
+   you give it, you can also provide a message to output when the test fails.
+   There is nothing as shit as when you have 1 test out 100 failing and 
```

Comment:
I don't think I've ever heard the expression "nothing as shit", which makes it seem unsuitable for a sample test. ;)